### PR TITLE
docs: update basic pages

### PR DESCRIPTION
**Pull request by kv-chiu**

### CODE REVIEW

**Comment by hsluoyz** [comment]
@Selflocking

File: sidebars.js [hunk 1]
```diff
@@ -19,8 +19,18 @@ module.exports = {
...
+        },
+        {
+          type: "link",
+          label: "Server Installation",
+          href: "/docs/basic/server-installation",
+        },
+        {
+          type: "link",
+          label: "Casdoor SSO",
+          href: "/docs/basic/casdoor-sso",
```
Comment (Selflocking) [inline sidebars.js:33]: You can define sidebar like this:

```js
module.exports = {
  tutorialSidebar: [
    {
      type: "category",
      label: "The Basics",
      link: { type: "generated-index" },
      items: [
        "overview",
        "basic/core-concepts",
        "basic/server-installation",
        "basic/casdoor-sso",
      ],
    },
  ],
};
```

For more information, please visit https://docusaurus.io/docs/sidebar/items

**Review by Selflocking — CHANGES_REQUESTED** [review]

File: sidebars.js (hunk 1 above)
Comment (Selflocking) [reply sidebars.js]: It works fine on my machine, and the [preview](https://deploy-preview-7--casibase.netlify.app/docs/basic/core-concepts/) works fine too.

You can run `yarn clear` and try again

Comment (kv-chiu) [reply sidebars.js]: That worked perfectly! Thank you so much for your assistance!

**Pull request merged**